app/models/spree/product.rb

```diff
@@ -304,6 +304,16 @@ def variant_unit_with_scale=(variant_unit_with_scale)
       )
     end
 
+    # Remove any unsupported HTML.
+    def description
+      Rails::HTML::SafeListSanitizer.new.sanitize(super)
```

Comment:
I am wondering if we should add this to our `HtmlSanitizer` class ? ie something like : `HtmlSanitizer.rails_sanitize` 
The idea is to have all the sanitizer in the same place so we can pick and choose the appropriate one, instead of calling various different sanitizer in various places.
@openfoodfoundation/core-devs, any though ?

Reply:
Agreed. In fact, if we can assume SafeListSanitizer is safe, then I  don't see why we have to specify an allowlist at all. 

Perhaps Maikel took the opinion of allowing only what is needed. In that case, I think we should keep one sanitising method, and add any elements necessary for ActionText to our `HtmlSanitizer`.

@mkllnk can you confirm your thoughts?

Reply:
Please proof me wrong but I thought that the default list can still be dangerous. For example, I think that it allows the `class` attribute. Some CSS classes are used by Javascript to add logic to elements. So suddenly someone could create a link that looks like a button in our design. It could be called _Checkout_ and lead to a fake payment site.

Does the editor only rely on HTML comments? We could probably allow those.

Reply:
Good point, we should add a case for that to `html_sanitizer_spec`, then we can prove it, right or wrong 😉

Reply:
Great discussion! I've updated our custom sanitizer with the tags that are currently supported by our trix editor, and a specific `data-trix` attribute that is needed for the horizontal rule.

I've also added a more complete set of dangerous scenarios to the tests. This serves both as documentation of potential risks and as a proactive warning system through test failures should any new dangerous tags be introduced in the future.